### ISSUE TITLE
no use auxiliary_loss in cGAN

### ISSUE THREAD

**Issue by zhangxiaodi** [eriklindernoren/PyTorch-GAN#37]
hi, i am wonder in cgan. the auxiliary_loss is not use in optimizer_G.step(). But the CGAN can training normal and get correct result. I think, Maybe i overlook some significant details. So who can give me some tips. thanks !

**Comment by eriklindernoren** [comment]
Hey,

You're right, there is no explicit loss forcing the generator to produce samples of their corresponding labels. But since the discriminator is fed both the images and labels, the generator will have to produce digits corresponding to their respective labels in order to fool the discriminator. 

Cheers!
